app/panel/components/Summary.jsx

```diff
@@ -219,15 +220,24 @@ class Summary extends React.Component {
 		this.props.history.push(this._isPlusSubscriber() ? '/subscription/info' : `/subscribe/${!!this.props.user}`);
 	}
 
+	/**
+	 * Show the Rewards view
+	 * Used to handle user clicking on the Rewards icon
+	 */
+	showRewardsView() {
+		// this.toggleExpert('gotorewards');
+		this.props.actions.toggleExpert();
+	}
+
 	/**
 	 * Toggle between Simple and Detailed Views.
 	 */
-	toggleExpert() {
+	toggleExpert(subview = 'blocking') {
```

Comment:
With this default, maybe you don't need to change line 125.

Reply:
Unnecessary parameter removed from toggleExpert call in clickDonut